src/main/java/com/wanderersoftherift/wotr/core/rift/RiftEvents.java

```diff
@@ -22,9 +23,11 @@ public static void onRiftTick(LevelTickEvent.Post event) {
             for (var playerID : data.getPlayers()) {
```

Comment:
We should avoid making super long tick events. The player check should probably move into a separate method.

In general, I also feel like this whole tick method should probably also move to something like RiftData. And the event class should call that directly. We will likely get a lot of stuff in this tick method eventually and some children that need to be called and similar.

Reply:
I have split the post tick event handler into a PlayerChangedDimensionEvent handler and a LivingDeathEvent handler. I've also moved more of the logic into RiftLevelManager (I felt that was the better place to encapsulate the logic, with RiftData mostly managing the data of which players are in the rift).